src/gc/collector.cpp

```diff
@@ -283,30 +302,47 @@ void GCVisitor::visitPotentialRange(void* const* start, void* const* end) {
     }
 }
 
-static int ncollections = 0;
-
-void markPhase() {
-#ifndef NVALGRIND
-    // Have valgrind close its eyes while we do the conservative stack and data scanning,
-    // since we'll be looking at potentially-uninitialized values:
-    VALGRIND_DISABLE_ERROR_REPORTING;
-#endif
+static inline void visitByGCKind(void* p, GCVisitor& visitor) {
```

Comment:
I do worry that the compiler will choose at some point in the future to *not* inline function, even with additional specifiers/attributes/etc.  Not sure how we can ensure that without some post processing, though :/

For now I've verified that it does get inlined, and we can deal with the non-inline case later if it pops up (since I'm pretty sure it'll cause a perf regression if it does.)